python/cutlass/utils/lazy_import.py

```diff
@@ -0,0 +1,11 @@
+import importlib
+from typing import Any
+
+def lazy_import(mod_name: str) -> Any:
```

Comment:
Use a TypeVar for the return, even then I am not sure whether or not this destroys typing info about what module is imported...